commcare_connect/static/css/tailwind.css

```diff
@@ -1,15 +1,368 @@
 @import "tailwindcss";
+@tailwind utilities;
 
-
+[x-cloak] {
+    display: none !important;
+}
 @theme {
-  --color-brand-deep-purple: #16006D;
-  --color-brand-indigo: #3843D0;
-  --color-brand-cornflower-blue: #5D70D2;
-  --color-brand-sky: #8EA1FF;
-  --color-brand-sunset: #E44434;
-  --color-brand-mango: #FC5F36;
-  --color-brand-marigold: #FEAF31;
-  --color-brand-blue-light: #94A3B8;
-  --color-brand-sunset-light: #FC5F3633;
-  --color-brand-border-light: #E2E8F0;
+    --font-sans : "Work Sans", "sans-serif"; 
+    --breakpoint-sm:5rem;
+    --color-brand-deep-purple : #16006D;
+    --color-brand-indigo : #3843D0;
+    --color-brand-cornflower-blue : #5D70D2;
+    --color-brand-sky : #8EA1FF;
+    --color-brand-sunset : #E44434;
+    --color-brand-mango : #FC5F36;
+    --color-brand-marigold : #FEAF31; 
+    --color-brand-blue-light : #94A3B8;
+    --color-brand-sunset-light : #FC5F3633;
+    --color-brand-border-light : #E2E8F0; 
+}
+
+/* ------ Typography------ */
+
+/* display */
+.text-display-lg{
+    @apply text-6xl;   
+}
+.text-display-md{
+    @apply text-6xl;   
+}
+.text-display-sm{
+    @apply text-6xl;   
+}
+
+/* headline */
+.text-headline-lg{
+    @apply text-6xl;   
+}
+.text-headline-md{
+    @apply text-6xl;   
+}
+.text-headline-sm{
+    @apply text-6xl;   
+}
+
+/* title */
+.text-title-lg{
+    @apply text-6xl;   
+}
+.text-title-md{
+    @apply text-6xl;   
+}
+.text-title-sm{
+    @apply text-6xl;   
+}
+/* body */
+.text-body-lg{
+    @apply text-6xl;   
+}
+.text-body-md{
+    @apply text-6xl;   
+}
+.text-body-sm{
+    @apply text-6xl;   
+}
+/* label */
+
+.text-label-lg{
+    @apply text-6xl;   
+}
+
+.text-label-lg-heavy{
+    @apply text-6xl;   
+}
+
+.text-label-md{
+    @apply text-6xl;   
+}
+
+.text-label-md-heavy{
+    @apply text-6xl;   
+}
+
+.text-label-sm{
+    @apply text-6xl;   
+}
+ 
+/*-- badges --------------------------------------*/
+.badge {
```

Comment:
Should these components be declared inside @layer so that these classes can be part of other components?